docs/v1/use-cases.md

```diff
@@ -1,96 +1,143 @@
-## V1 Use Cases
+# Service Catalog Use Cases
 
-This document contains the complete list of accepted use-cases for the v1
-version of the service catalog.
+## Catalog Management
 
-## High-Level Use Cases
+1. As a user, I want to be able to register a broker with the Kubernetes service
```

Comment:
@duglin - I think we should refine 'user' throughout, but probably not in this PR.